packages/react-native-reanimated/Common/cpp/reanimated/RuntimeDecorators/REAWorkletRuntimeDecorator.cpp

```diff
@@ -0,0 +1,14 @@
+#include "REAWorkletRuntimeDecorator.h"
+#include "JSISerializer.h"
+#include "PlatformLogger.h"
+#include "ReanimatedJSIUtils.h"
+#include "WorkletRuntime.h"
+
+namespace reanimated {
```

Comment:
Add newline
```suggestion
namespace reanimated {

```

Reply:
All of this code was through our formatters. Seems like we are missing some settings?

Reply:
I've been through clang-format config options and there's no option for this specific case. I can add a newline here manually but I don't think it's something we should enforce in the codebase.